### PR TITLE
feat(kyve): init KYVE crypto fund PE-6448

### PR DESCRIPTION
**Pull request by fedellen**
This PR introduces KYVE crypto fund to the turbo SDK

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "@kyvejs/sdk": "^1.3.2",
     "@solana/web3.js": "^1.91.7",
     "arbundles": "0.9.9",
-    "axios": "^1.4.0",
+    "axios": "1.4.0",
```
Comment (fedellen) [inline package.json:82]: seeing some type issues with axiosRetry when this is pulling in a latest version 🤔

Comment (dtfiedler) [reply package.json:82]: I'll take a lookin- we resolved this in ar-io-sdk

File: src/global.d.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+interface Window {
```
Comment (dtfiedler) [inline src/global.d.ts:19]: This should likely only exist for web, but fine for now

**Review by dtfiedler — APPROVED** [review]

**Pull request merged**